src/options/containers/notifications/index-pouch.js

```diff
@@ -0,0 +1,27 @@
+import PouchDB from 'pouchdb'
+
+const db = new PouchDB('notifs')
```

Comment:
I think this is just for testing, but as we already have a PouchDB instance in the project, that should be the one imported everywhere and used instead. So instead of importing from this module, all the other modules that need to query PouchDB should have `import db from 'src/pouchdb'` at the top so they use the main DB instance.